views/layouts/layout-redirect.njk

```diff
@@ -0,0 +1,20 @@
+---
+layout: layout-single-page.njk
+ignore_in_sitemap: true
```

Comment:
Unfortunately it doesn't look like this gets merged into the frontmatter for the page – the cookies page still appears in the sitemap